learning_resources_search/serializers.py

```diff
@@ -336,6 +362,11 @@ def _transform_search_results_suggest(search_result):
         return []
 
 
+class SearchResponseMetadata(TypedDict):
+    aggregations: dict[str, list[AggregationValue]]
+    suggestions: list[str]
```

Comment:
@abeglova Can you double check that this metadata schema (this, and the generated one) look correct?